tests/functional/pages/features/calculated_summary/currency-total-playback.page.js

```diff
@@ -11,46 +11,68 @@ class CurrencyTotalPlaybackPage extends QuestionPage {
 
   firstNumberAnswerEdit() { return '[data-qa="first-number-answer-edit"]'; }
 
+  firstNumberAnswerLabel() { return '[data-qa="first-number-answer-label"]'; } // Field not generated with generated with generate_pages.py script
```

Comment:
I feel this is a slippery path to head down.
These files are generated, as soon as we start making changes that are not generated then we are opening ourselves up for a world of pain?